### PR TITLE
add good looking error pages

### PR DESCRIPTION
**Pull request by mbernasocchi**
This PR adds good looking error 403, 404 and 500 error pages.

it includes a minimal css and a new svg. Feel free to refactor css and svg location to wherever it is ideal. Ive not much clue of where things should ideally be[e] :)

image

### CODE REVIEW

File: conf/nginx/pages/403.html [hunk 1]
```diff
@@ -1,12 +1,21 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>403 Forbidden</title>
+    <title>Error 404 - Forbidden </title>
```
Comment (suricactus) [inline conf/nginx/pages/403.html:7]:
```suggestion
    <title>Error 403 - Forbidden </title>
```
Isn't it?

File: conf/nginx/pages/403.html [hunk 2]
```diff
@@ -1,12 +1,21 @@
...
+    <link rel="stylesheet" href="style.css">
 </head>
-<body bgcolor="white">
-    <center><h1>403 Forbidden</h1></center>
+
+<body>
+    <div class="container text-center">
+        <img src="sad_nyuki.svg" alt="Sad Nyuki" class="mt-5 mb-4">
+        <h1 class="display-4">404</h1>
```
Comment (suricactus) [inline conf/nginx/pages/403.html:14]:
```suggestion
        <h1 class="display-4">403</h1>
```

**Review by suricactus — COMMENTED** [review]
Ideally I would try to avoid import of bootstrap like this.


We can inline the CSS in each of the files. Not a strong opinion, but my argument here would be - reduce the external dependencies on error pages, there was already something wrong, dependencies might be broken.

**Comment by faebebin** [comment]
ok, good point. will inline the css

**Pull request merged**